include/guisan/widgets/dropdown.hpp

```diff
@@ -150,7 +150,7 @@ namespace gcn
          * @return the ListModel used.
          * @see setListModel
          */
-        ListModel *getListModel();
+        ListModel *getListModel() const;
```

Comment:
It is a case which is "unclear" as pointer doesn't propagate `const`.
Having both
```
const ListModel *getListModel() const;
ListModel *getListModel();
```
might be better.

Apply to other places too.